docs-compose/overwatch_docs/content/GettingStarted/Configuration.md

```diff
@@ -214,4 +217,28 @@ Config | Required Override | Default Value | Type | Description
 **enabled**|N|false|Double|Approximate list price of interactive DBU
 **minimumCores**|N|4|Int|Minimum number of cores to be used during Overwatch run
 **maximumCores**|N|512|Int|Maximum number of cores to be used during Overwatch run
-**coeff**|N|1.0|Double|Scaler, each module has a scale based on it's size relative to the other modules. This variable acts as a scaler to the scaler, if the modules are scaling too fast (or not fast enough), this can be tweaked to increase the variability of the scaling from the starting core count.
\ No newline at end of file
+**coeff**|N|1.0|Double|Scaler, each module has a scale based on it's size relative to the other modules. This variable acts as a scaler to the scaler, if the modules are scaling too fast (or not fast enough), this can be tweaked to increase the variability of the scaling from the starting core count.
+
+### ApiEnvConfig
+Configurations of Api calls.
+
+Config | Required Override | Default Value | Type | Description
+:--------------------------|:---|:----------|:----------|:--------------------------------------------------
+**successBatchSize**|N|200|Int|Responce of successful api calls are stored into the disk.This config signifies the batch size.
+**errorBatchSize**|N|500|Int|Failed api calls are stored into disk.This config signifies the batch size of failed api calls.
+**enableUnsafeSSL**|N|false|Boolean|Enable unasfe SSL.
+**threadPoolSize**|N|4|Int|Number of threads to perform api calls in parallel.
+**apiWaitingTime**|N|300000|milliseconds|Api waiting time incase of delay in response.
+**apiProxyConfig**|N|None|Option[\[ApiProxyConfig\]](#apiproxyconfig)|Allows user to configure proxy.
```

Comment:
default value is ApiProxyConfig() not none

Reply:
done